test/rules/jsx-whitespace-literal/test.tsx.lint

```diff
@@ -0,0 +1,29 @@
+<div>
+    <p>Some Text</p>
+    <hr></hr>
+    Some Text&nbsp;&nbsp;
```

Comment:
the failure should only be on the `&nbsp;&nbsp;` range

Reply:
Done